spec/models/user_spec.rb

```diff
@@ -1,5 +1,29 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  let!(:user) { User.new(first_name: "Ashley", last_name: "Hampson", email: "Ashley.Hampson@gmail.com", password: "password") }
```

Comment:
Instead of checking to make sure that each property returns the right value, you want to focus this testing on determining whether your validations are working correctly. Consider using `shoulda-matchers` to do so! 

```

Reply:
oops, meant to include this example:

  describe 'validations' do
    it { should validate_presence_of(:first_name) }
    it { should validate_presence_of(:last_name) }
    it { should validate_presence_of(:email) }
  end

Reply:
We made a commit with fixes. Let us know if anything else is needed!